runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/ProcessBundleDescriptors.java

```diff
@@ -179,9 +298,13 @@ private static String addWireCoder(
     public static ExecutableProcessBundleDescriptor of(
         ProcessBundleDescriptor descriptor,
         RemoteInputDestination<WindowedValue<?>> inputDestination,
-        Map<BeamFnApi.Target, Coder<WindowedValue<?>>> outputTargetCoders) {
+        Map<BeamFnApi.Target, Coder<WindowedValue<?>>> outputTargetCoders,
+        Map<String, Map<String, MultimapSideInputSpec>> multimapSideInputSpecs) {
       return new AutoValue_ProcessBundleDescriptors_ExecutableProcessBundleDescriptor(
-          descriptor, inputDestination, Collections.unmodifiableMap(outputTargetCoders));
+          descriptor,
+          inputDestination,
+          Collections.unmodifiableMap(outputTargetCoders),
+          multimapSideInputSpecs);
```

Comment:
Should we be consistent in making defensive views (or not doing so)?

Reply:
done